tensorflow-core/tensorflow-core-api/src/bazel/op_generator/op_generator.cc

```diff
@@ -349,6 +349,10 @@ void RenderGettersAndSetters(const OpSpec& op, SourceWriter* writer) {
         .EndMethod();
   }
   for (const ArgumentSpec& output : op.outputs()) {
+    if ("size" == output.var().name()) {
+        // reserved name
+        continue;
+    }
```

Comment:
We should not skip the getter of an output that have a reserved name but just rename it in the API def, though I'm not sure why `size` would cause any problem here. Can you give more context to this change?

Reply:
There are new ops with outputs named "size". Do whatever you want with them, I just did that to get them to compile.

Reply:
Ok, I'll take care of it